playground/TestCommon/TestArrayMaze.cs

```diff
@@ -0,0 +1,84 @@
+using Common;
+
+namespace TestCommon;
+
+public class TestArrayMaze
+{
+    [Test]
+    [TestCase(5, 5, 5)]
+    [TestCase(1, 2, 3)]
+    [TestCase(3, 2, 1)]
+    [TestCase(1, 10, 50)]
+    [TestCase(12, 23, 44)]
+    public void TestConstructor(int sizeX, int sizeY, int blockSize)
+    {
+        //Arrange    
+        var random = new Random(sizeX + sizeY + blockSize);
+        var storage = new[] { random.NextInt64() };
+
+        //Act
+        var maze = new ArrayMaze(sizeX, sizeY, blockSize, storage);
+
+        //Assert
+        Assert.Multiple(() =>
+        {
+            //Assert
```

Comment:
remove this

Reply:
Why?
Isn't it right to have it? 
If 2 assets fail it will show both and not stop at first
<img width="721" alt="image" src="https://github.com/kiIIer/mAIze/assets/39669501/ba2a7a25-3999-4048-b5e0-6f539ec4ffcc">

Or if you are about 2nd comment, yeah I'll remove that